src/app/application.cpp

```diff
@@ -1157,7 +1157,7 @@ void Application::applyMemoryWorkingSetLimit() const
         }
         LogMsg(logMessage.arg(QString::number(errorCode), message), Log::WARNING);
     }
-#elif defined(Q_OS_UNIX)
+#elif defined(Q_OS_UNIX) && !defined(Q_OS_MACOS)
```

Comment:
Shouldn't you entirely remove this option on macOS? In the same way as it doesn't exist in libtorrent-1.2 based builds?

Reply:
Done.

Reply:
So maybe also reword commit message?